cmd/user-agent/src/store/modules/mediator.js

```diff
@@ -9,21 +9,16 @@ import {waitForEvent, POST_STATE} from '../../events'
 
 const stateCompleted = 'completed'
 const topicDidExchangeStates = 'didexchange_states'
-const errRouterNotRegistered = 'router not registered'
 
 export default {
     actions: {
         async loadMediatorState({commit, getters}) {
             let aries = getters['aries/getInstance']
-            let res = await aries.mediator.getConnection().catch(err => {
-                if (!err.message.includes(errRouterNotRegistered)) {
-                    throw err
-                }
-            })
+            let res = await aries.mediator.getConnections()
 
             let connectionID = '';
-            if (res) {
-                connectionID = res.connectionID
+            if (res.connections) {
+                connectionID = res.connections[0]
```

Comment:
If the user is registered with several mediators, then their "mediator state" is not just defined by the first one one the list that happens to be returned in a particular API call.

If the user has registered several mediators then there should be a view that shows them all. That single "Mediator" box in the `Connections` form needs rework. For instance, how would the `unregister` button work with multiple?

Reply:
I assume there is a plan to start incorporating the idea that the user can have multiple routers, and that the UI elements will be adjusted accordingly.

Reply:
Is there a plan to display the list of mediators? Currently the UI will misinform the user:

![image](https://user-images.githubusercontent.com/2019896/95450440-4bd2fd00-0934-11eb-8d3d-ad3a5d220af4.png)